frontend/src/hooks/useFormData/index.tsx

```diff
@@ -164,6 +188,13 @@ const useFormData = (): [FormData, Meta, Helpers] => {
     useControlledClearEffect(formData.didFellowshipReachMordor.value, "mordor");
     useControlledClearEffect(formData.wasAragornCrowned.value, "aragornTurn");
 
+    useStrongholdDeselectEffect(
```

Comment:
I'm not sure the best way to organize this stuff, but the general design principle would be that (as much as is possible) we want a single source of truth for the information.

I wonder if the types could be simplified (just have one single type: `Settlement`, representing all cities/strongholds), and then this could be encoded somewhere in a single function? The backend has a function like this:

```hs
strongholdSide :: [Expansion] -> Stronghold -> Side
strongholdSide expansions stronghold
  | FateOfErebor `elem` expansions && stronghold == Erebor = Shadow
  | otherwise = case stronghold of
      Rivendell -> Free
      GreyHavens -> Free
      WoodlandRealm -> Free
      Lorien -> Free
      HelmsDeep -> Free
      Edoras -> Free
      MinasTirith -> Free
      DolAmroth -> Free
      Pelargir -> Free
      Shire -> Free
      Dale -> Free
      Erebor -> Free
      EredLuin -> Free
      IronHills -> Free
      MountGundabad -> Shadow
      Angmar -> Shadow
      Moria -> Shadow
      DolGoldur -> Shadow
      Orthanc -> Shadow
      Morannon -> Shadow
      BaradDur -> Shadow
      MinasMorgul -> Shadow
      Umbar -> Shadow
      FarHarad -> Shadow
      SouthRhun -> Shadow
  ```
  
  Alternatively possible useful function sigs could be:
  `canCaptureSettlement :: Side -> [Expansion] -> Settlement -> Bool`

Reply:
Used this as a guide to update, it's a lot better now 💯 not sure if it's all the way there yet though, switched to discord for brainstormy stuff

Reply:
Nvm - pushed one more improvement and now I'm pretty happy with it, it's ready for another look